docker/gcp_pudl_etl.sh

```diff
@@ -208,9 +208,20 @@ function merge_tag_into_branch() {
 
 function clean_up_outputs_for_distribution() {
     # Compress the SQLite DBs for easier distribution
-    gzip --verbose "$PUDL_OUTPUT"/*.sqlite && \
-    # Grab hourly tables which are only written to Parquet for distribution
-    cp "$PUDL_OUTPUT"/parquet/*__hourly_*.parquet "$PUDL_OUTPUT" && \
+    cd "$PUDL_OUTPUT" && \
+    for file in *.sqlite; do
+        echo "Compressing $file" && \
+        zip "$file.zip" "$file" && \
+        rm "$file"
+    done
```

Comment:
I thought what was being ANDed together was the exit codes? So if they're all 0, then the exit code for the whole chain of codes ends up being 0, but if any of them is non-zero, then the remaining commands are not executed, and the exit code ends up being whatever nonzero value the failing command issued?

Reply:
Yeah, I think the exit codes are the things that are being ANDed together. We're using the short-circuiting behavior of `&&` to say that each expression only executes if everything to its left returns 0.

But we have several separate `&&`-chains, and we don't exit immediately if one of the chains themselves has a non-0 return value. I think that's intentional so we can make the slack notifications at the end.

The body of the for loop is its own `&&`-chain because we don't have `&&` after `rm "$file"` - so in my mental model, a failure in `zip` would stop us from trying to `rm "$file"` but we would effectively just skip over that file completely.